CMakeLists.txt

```diff
@@ -117,8 +117,12 @@ add_subdirectory(benchmark)
 if (${BPFTIME_BUILD_STATIC_LIB}) 
 message(STATUS " Adding libraries to static archive")
 bpftime_add_static_lib_component_command(runtime)
-bpftime_add_static_lib_component_command(attach)
-bpftime_add_static_lib_component_command(vm)
+bpftime_add_static_lib_component_command(base_attach_impl)
+bpftime_add_static_lib_component_command(frida_uprobe_attach_impl)
+bpftime_add_static_lib_component_command(syscall_trace_attach_impl)
+bpftime_add_static_lib_component_command(text_segment_transformer)
```

Comment:
I think we may not need to package text_segment_transformer into the static target?

Reply:
yea you are right, It was generating a shared library, I was getting an error when I tried to package it using `ar`, Also, I wll create a new PR, closing this one, cuz it is a lot behind the master branch, will create a new branch from the master and send PR from that.